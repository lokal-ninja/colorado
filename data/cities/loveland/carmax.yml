id: 4469335512
type: node
latitude: 40.43014
longitude: -104.9945
postcode: "80538"
city: Loveland
street: Byrd Drive
housenumber: "3760"
phone: ""
opening_hours: ""
website: https://www.carmax.com/stores/7258
