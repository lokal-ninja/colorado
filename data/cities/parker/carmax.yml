id: 634178076
type: way
latitude: 39.53701
longitude: -104.77671
postcode: "80134"
city: Parker
street: "Ponderosa Drive"
housenumber: 18220
phone: ""
opening_hours: ""
website: ""