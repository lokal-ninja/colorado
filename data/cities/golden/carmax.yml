id: 913228638
type: way
latitude: 39.7228403
longitude: -105.1863725
postcode: "80401"
city: Golden
street: "West Colfax Avenue"
housenumber: 16500
phone: "+1 303-278-0152"
opening_hours: ""
website: "carmax.com"
